x-pack/plugins/security/server/authorization/privileges/privileges.test.ts

```diff
@@ -835,19 +692,13 @@ describe('reserved', () => {
     const actual = privileges.get();
     expect(actual).toHaveProperty('reserved.foo', [
       actions.version,
-      actions.app.get('app-1'),
-      actions.app.get('app-2'),
-      actions.ui.get('catalogue', 'catalogue-1'),
-      actions.ui.get('catalogue', 'catalogue-2'),
-      actions.ui.get('management', 'foo', 'management-1'),
-      actions.ui.get('management', 'foo', 'management-2'),
       actions.ui.get('navLinks', 'kibana:foo'),
     ]);
   });
 
   test('actions defined at the reservedPrivilege take precedence', () => {
```

Comment:
Do we need this test anymore now that we aren't cascading?

Reply:
Nope!